Added missing field `typeName` to the mapping file for resources.

praves77 opened the pull request.

maryhowell commented on config/mapping_resources.json:
```diff
@@ -2,15 +2,35 @@
+            "type": "keyword",
             "index": true
         },
-        "experimentId": {
```
I think we still need the experimentId as well.

praves77 commented:
@maryhowell I added you too - as I saw your comment on my last PR. In the previous PR, I had just changed how id fields would get indexed/searched and that would not have any impact their presence in search results. They would  be always there in the searched response.
